src/ducks/field-guide.js

```diff
@@ -0,0 +1,51 @@
+import apiClient from 'panoptes-client/lib/api-client.js';
+import { config } from '../config';
+
+const FETCH_GUIDE = 'FETCH_GUIDE';
+
+const initialState = {
+  icons: null,
+  guide: null
```

Comment:
One thing to note is that the Field Guide doesn't follow the standard `status: idle/fetching/success/error` pattern of our other ducks.

On one hand, this will make understanding the state of the Field Guide - e.g. for telling the user the status of the guide - more difficult.

On the other hand, the Field Guide is a low enough on the priority list that I won't make too much of an issue for it. Again, this is something we can improve in the future - we just need the MVP for the beta test at the moment.

Reply:
Nah, I need to start doing this as some API calls are getting murky in their dependencies of other resources. I'll try to make the change pretty quickly so I can get comfortable with the approach and apply it to other items.